### ISSUE TITLE
Improve alt text for logos on About Us page

### ISSUE THREAD

**Issue by carrythebanner** [calagator/calagator.org#29]
On the "About us" page (http://calagator.org/about), the "Attributions" section credits a number of logo images. Each image has alt text, but currently it's a mix of a useful label and some random letters and numbers. For example: 
`<img src="/assets/tag_icons/ruby-6cb4bb736e6c4cb0000f6e4d6254e13d9088263198adc4a1fd0676f8a2f84c34.png" alt="Ruby 6cb4bb736e6c4cb0000f6e4d6254e13d9088263198adc4a1fd0676f8a2f84c34">`

Alt text should be human-readable and contextually appropriate. Images will be identified as such by screen readers, so typically alt text should **not** include the word "image." But in this case since logos are specifically being identified, I'd recommend "Ruby logo" (vs. just "Ruby"), as in: 
`<%= image_tag "tag_icons/ruby.png" alt: "Ruby logo" %>`
